elide-async/src/test/java/com/yahoo/elide/async/service/AsyncQueryThreadTest.java

```diff
@@ -77,8 +158,10 @@ public void testProcessQueryGraphQl() throws Exception {
         AsyncQueryThread queryThread = new AsyncQueryThread(queryObj, user, elide, runner, asyncQueryDao, "v1",
                 resultStorageEngine);
         queryResultObj = queryThread.processQuery();
-        assertEquals(queryResultObj.getResponseBody(), "ResponseBody");
+        assertEquals(queryResultObj.getRecordCount(), 3);
+        assertEquals(queryResultObj.getResponseBody(), responseBody);
         assertEquals(queryResultObj.getHttpStatus(), 200);
+
     }
 
```

Comment:
Please include some negative test cases for jsontocsv convert method, failure scenarios